internal/pkg/describe/env.go

```diff
@@ -0,0 +1,104 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package describe
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/archer"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/aws/session"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/store"
+	"github.com/aws/aws-sdk-go/service/cloudformation"
+)
+
+type EnvironmentSummary struct {
```

Comment:
Just curious - why not use the Environment struct in `archer/Environment`?

Reply:
Hmm. describe/project.go defines its own project and application structs--I had moved EnvironmentSummary over from project.go (and renamed it to distinguish it from my larger Environment struct. Is it because we don't use all of the available fields?

Reply:
After a discussion with @efekarakus, I'm going to refactor to use archer's structs, for both apps and envs!